### ISSUE TITLE
no calendar view [IE11]

### ISSUE THREAD

**Issue by ShinjiLE** [nextcloud/calendar#1957]
### Steps to reproduce
1.go to the calendar view
2.
3.

### Expected behaviour
The grid and the sidepanel should show

### Actual behaviour
only the mainpanel on the top is loaded .

### Calendar app
**Calendar app version:** 2.0.1

### Client configuration
**Browser:** IE 11.0.101

**Operating system:** Win7

### Server configuration

**Operating system**: Debian Sid

**Web server:** Apache

**Database:** MariaDB

**PHP version:**  7.0.3-fpm

**Nextcloud Version:** 18.0.0

**Updated from an older installed version or fresh install:** updated

**List of activated apps:**

```
Enabled:
  - activity: 2.11.0
  - admin_audit: 1.8.0
  - bookmarks: 2.3.4
  - bruteforcesettings: 1.5.0
  - calendar: 2.0.1
  - checksum: 0.4.4
  - circles: 0.18.1
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - contacts: 3.1.8
  - dav: 1.14.0
  - event_update_notification: 1.0.1
  - extract: 1.2.3
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_accesscontrol: 1.8.1
  - files_automatedtagging: 1.8.0
  - files_external: 1.9.0
  - files_fulltextsearch: 1.4.1
  - files_fulltextsearch_tesseract: 1.4.0
  - files_linkeditor: 1.0.11
  - files_markdown: 2.2.0
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trackdownloads: 1.7.0
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - files_videoplayer: 1.7.0
  - firstrunwizard: 2.7.0
  - flowupload: 0.1.8
  - fulltextsearch: 1.4.1
  - fulltextsearch_elasticsearch: 1.5.0
  - groupfolders: 6.0.1
  - impersonate: 1.5.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - mail: 1.1.2
  - maps: 0.1.4
  - music: 0.12.1
  - news: 14.1.2
  - nextbackup: 19.12.1
  - nextcloud_announcements: 1.7.0
  - notes: 3.1.1
  - notifications: 2.6.0
  - oauth2: 1.6.0
  - passman: 2.3.5
  - password_policy: 1.8.0
  - phonetrack: 0.6.1
  - photos: 1.0.0
  - previewgenerator: 2.2.0
  - pride: 0.0.3
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - recommendations: 0.6.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - spreed: 8.0.1
  - support: 1.1.0
  - survey_client: 1.6.0
  - suspicious_login: 3.0.1
  - systemtags: 1.8.0
  - tasks: 0.11.3
  - text: 2.0.0
  - theming: 1.9.0
  - twofactor_backupcodes: 1.7.0
  - updatenotification: 1.8.0
  - user_ldap: 1.8.0
  - viewer: 1.2.0
  - w2g2: 3.0.1
  - workflow_script: 1.3.1
  - workflowengine: 2.0.0
Disabled:
  - accessibility
  - analytics
  - apporder
  - encryption
  - workflow_pdf_converter

```

**Nextcloud configuration:**

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.mybase.selfhost.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "tempdirectory": "\/var\/tmp\/nextcloudtemp",
        "overwrite.cli.url": "https:\/\/nextcloud.mybase.selfhost.de",
        "knowledgebaseenabled": true,
        "dbtype": "mysql",
        "version": "18.0.0.10",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "updater.release.channel": "beta",
        "appstore.experimental.enabled": true,
        "maintenance": false,
        "loglevel": 3,
        "debug": false,
        "htaccess.RewriteBase": "\/",
        "enable_previews": true,
        "preview_max_x": 1080,
        "preview_max_y": 1920,
        "preview_max_filesize_image": 100,
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "preview_office_cl_parameters": "--headless --nologo --nofirststartwizard --invisible --norestore --convert-to pdf --outdir ",
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\TIFF",
            "OC\\Preview\\SVG",
            "OC\\Preview\\Postscript",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\PDF",
            "OC\\Preview\\Font",
            "OC\\Preview\\Epub",
            "OC\\Preview\\StarOffice",
            "OC\\Preview\\TIF"
        ],
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "log_type": "syslog",
        "logfile": "\/var\/log\/nextcloud.log",
        "logfilemode": 416,
        "log_rotate_size": 104857600,
        "syslog_tag": "Nextcloud",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "filelocking.enabled": "true",
        "filelocking.ttl": 3600,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 0,
            "dbindex": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "memcached_servers": [
            [
                "127.0.0.1",
                11211
            ]
        ],
        "data-fingerprint": "790e9a77b6ee76b2af214c8f58d75dfd",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mysql.utf8mb4": true,
        "allow_user_to_change_display_name": true,
        "auth.bruteforce.protection.enabled": true,
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "theme": ""
    }
}

```

#### Web server error log (e.g. /var/log/apache)
```
Insert your webserver log here
```

#### Log file (data/nextcloud.log)
```
Insert your nextcloud.log file here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) SCRIPT1002 Syntaxfehler 
calendar.js Line:2 R:1017678
```

**Comment by georgehrke** [comment]
@ShinjiLE Can you please provide a screenshot, thx

**Comment by ShinjiLE** [comment]
Hi,
Sorry for the late response. After some other troubles i migrated to Apache 2.4.41 and php 7.4.3 .
The behavior is the same . Screenshot attached .
![nc18 calendarview not shown](https://user-images.githubusercontent.com/16802516/75156312-27624180-5712-11ea-83b1-760417043d36.png)

**Comment by georgehrke** [comment]
Okay, so ...
I was able to reproduce that bug.

There are plenty of issues:
- To begin with the easy one: 
Some dependencies are not IE11 compatible, so we would have to enable babel for them.
That could be done by adding `color-convert` and `v-autosize` to https://github.com/nextcloud/calendar/blob/a6c64c79890417e97c7f9cf3ff41689aaa9b883c/webpack.common.js#L45

- But once that is done, it keeps complaining about other things like missing `XPathResult`.
- Probably more after that, can't check without fixing the `XPathResult` thingy.

**Comment by georgehrke** [comment]
While we technically aim at supporting IE, this is really low priority and there are more important bugs / features that will be worked on first.

If someone wants to look into this, i would be happy to give some pointers.

**Comment by tcitworld** [comment]
There is no point in supporting IE anymore.
